tensorflow/python/estimator/run_config.py

```diff
@@ -288,6 +289,11 @@ def _validate(property_name, cond, message):
             message='device_fn must be callable with exactly'
                     ' one argument "op".')
 
+  _validate('protocol',
+            lambda protocol: protocol is None or protocol == "grpc"
```

Comment:
Optional: Make this `lambda protocol: protocol in (None, "grpc", "grpc+verbs")`

Also, the docstring of [`server_lib.Server`](https://github.com/tensorflow/tensorflow/blob/23e26ec2a7521c924b2a9fc435c04761f97fe6ba/tensorflow/python/training/server_lib.py#L131) and the [`ServerDef`](https://github.com/tensorflow/tensorflow/blob/23e26ec2a7521c924b2a9fc435c04761f97fe6ba/tensorflow/core/protobuf/tensorflow_server.proto#L49) protocol buffer message suggest that the only valid value for protocol is "grpc". Should those docstrings be updated too?

Reply:
Done, and docstring update.

Thanks.